sdk/src/main/java/com/horizen/cryptolibprovider/Sc2scCircuit.java

```diff
@@ -0,0 +1,38 @@
+package com.horizen.cryptolibprovider;
+
+
+import com.horizen.certnative.WithdrawalCertificate;
+import com.horizen.librustsidechains.FieldElement;
+import com.horizen.merkletreenative.MerklePath;
+import com.horizen.sc2sc.CrossChainMessage;
+import com.horizen.sc2sc.CrossChainMessageHash;
+
+import java.util.List;
+
+public interface Sc2scCircuit {
+
+    int getMaxMessagesPerCertificate();
+
+    CrossChainMessageHash getCrossChainMessageHash(CrossChainMessage msg);
+
+    byte[] getCrossChainMessageTreeRoot(List<CrossChainMessage> messages);
+
+    MerklePath gerCrossChainMessageMerklePath(List<CrossChainMessage> messages, CrossChainMessage leaf);
```

Comment:
typo: `gerCrossChainMessageMerklePath` instead of `getCrossChainMessageMerklePath`